frontend/src/component/admin/instance-privacy/InstancePrivacySection.tsx

```diff
@@ -0,0 +1,211 @@
+    display: 'table',
+}));
+
+const StyledDataCollectionPropertyCell = styled('div')(() => ({
+    display: 'table-cell',
+}));
+
+interface IPrivacyProps {
+    title: string;
+    infoText: string;
+    concreteDetails: Record<string, string>;
+    enabled: boolean;
+    changeInfoText: string;
+    variablesText: string;
+    dependsOnText?: string;
+}
+
+interface IToolTipInstructionContentProps {
+    changeInfoText: string;
+    variablesText: string;
+    dependsOnText?: string;
+}
+
+const ToolTipInstructionContent = ({
+    changeInfoText,
+    variablesText,
+    dependsOnText,
+}: IToolTipInstructionContentProps) => {
+    return (
+        <StyledDescription>
+            <ToolTipDescriptionText>{changeInfoText}</ToolTipDescriptionText>
+
+            <ToolTipDescriptionCode>
+                <div>{variablesText}</div>
+            </ToolTipDescriptionCode>
+
+            <ConditionallyRender
+                condition={Boolean(dependsOnText)}
+                show={
+                    <ToolTipDescriptionText>
+                        {dependsOnText}
+                    </ToolTipDescriptionText>
+                }
+            />
+        </StyledDescription>
+    );
+};
+
+const ToolTipDescriptionCode = styled('code')(({ theme }) => ({
+    display: 'block',
+    color: theme.palette.text.primary,
+    backgroundColor: theme.palette.background.application,
+    fontSize: theme.fontSizes.smallerBody,
+    marginTop: theme.spacing(1),
+    padding: theme.spacing(1),
+    border: `1px solid ${theme.palette.divider}`,
+    borderRadius: theme.shape.borderRadius,
+    borderWidth: 1,
+    wordWrap: 'break-word',
+    whiteSpace: 'pre-wrap',
+    fontFamily: 'monospace',
+    lineHeight: 1.5,
+}));
+
+const ToolTipDescriptionText = styled('p')(({ theme }) => ({
+    color: theme.palette.text.primary,
+    fontSize: theme.fontSizes.smallBody,
+    marginTop: theme.spacing(1),
+}));
+
+export const InstancePrivacySection = ({
+    title,
+    infoText,
+    concreteDetails,
+    enabled,
+    changeInfoText,
+    variablesText,
+    dependsOnText,
+}: IPrivacyProps) => {
+    return (
+        <StyledContainer>
+            <StyledCardTitleRow>
+                <b>{title}</b>
+                <StyledDataCollectionBadge>
+                    <ConditionallyRender
+                        condition={enabled}
+                        show={
+                            <Badge color="success" icon={<CheckIcon />}>
+                                Data is collected
+                            </Badge>
+                        }
+                    />
+                    <ConditionallyRender
+                        condition={!enabled}
+                        show={
+                            <Badge color="neutral" icon={<ClearIcon />}>
+                                No data is collected
+                            </Badge>
+                        }
+                    />
```

Comment:
I'm wondering why you added another `ConditionallyRender` instead of adding an `elseShow` prop to the one above. Is there something I'm missing or just an oversight?